src/main/resources/static/scripts/scripts.js

```diff
@@ -0,0 +1,42 @@
+let navbar = document.querySelector("nav.navbar");
```

Comment:
this navbar can be const

Reply:
yup, thanks -> replaced some lets with consts